app/controllers/admin_controller.rb

```diff
@@ -49,17 +49,29 @@ def rate_limits
 
   private
     def github_rate_limit
-      limit_response = machine_user.rate_limit
-      unless limit_response.limit.nil?
-        reset_time = Time.at(limit_response.resets_at)
-        return {"remaining": limit_response.remaining, "limit": limit_response.limit, "reset": reset_time,
+      limit_response = Octokit_Wrapper::Octokit_Wrapper.machine_user.post '/graphql', { query: rate_limit_graphql_query }.to_json
+      if limit_response.data.present?
+        rateLimitInfo = limit_response.data.rateLimit
+        reset_time = Time.parse(rateLimitInfo.resetAt)
+        return {"remaining": rateLimitInfo.remaining, "limit": rateLimitInfo.limit, "reset": reset_time,
                 "until_reset": distance_of_time_in_words_to_now(reset_time), "info": "GitHub Machine User: " +
-                ENV['MACHINE_USER_NAME']}
+                limit_response.data.viewer.login}
       end
       {"error": "Rate limit API request failed. Try refreshing the page."}
     end
 
-    def machine_user
-      Octokit_Wrapper::Octokit_Wrapper.machine_user
+    def rate_limit_graphql_query
+      <<-GRAPHQL
+        query {
+          viewer {
+            login
+          }
+          rateLimit {
+            remaining
+            limit
+            resetAt
+          }
+        }
+      GRAPHQL
```

Comment:
So if I'm understanding correctly, then we no longer need to check the old rate limit because we don't use it anymore. Does that sound correct?

Reply:
That's correct yeah. The only GitHub jobs we have now are the org membership and update repo ones which both use GraphQL. Anything further will also be done in GraphQL so we don't need the old one.